Installing-and-Configuring/BootOptions.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: Configuring Boot Options
+permalink: /BootOptions/
+parent: Installing and configuring
+nav_order: 85
+---
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+
+Boot config files
+===
+
+A user has the option to make persistent modifications to the non-Qubes boot
+ process by creating one or more of the following files:
+
+| file | description |
+| ---- | ---- |
+| kexec_menu.txt | contains multiple options for parameters to the kexec command|
+| kexec_hashes.txt | a sha256sum file from within the respective boot directory |
+| kexec_iso_add.txt | a sh variable to override the standard ISO kernel argument
+ additions |
+| kexec_iso_remove.txt | a sh variable to override the standard ISO kernel
+ argument removals |
+| kexec_default.$N.txt | specifies the default kexec parameters corresponding to
+ the Nth menu option |
+| kexec_default_hashes.txt | a sha256sum file for the default entry kexec file
+ parameters |
+| kexec_rollback.txt | a sha256sum of the TPM counter contents in the tmp
+ directory |
+| kexec_key_devices.txt | contains a list of "device uuid" combos for all LUKS
+ devices to unlock |
+| kexec_key_lvm.txt | contains the name of an LVM group to activate on boot |
+
```

Comment:
Plese review table https://jtmoree-github-com.github.io/heads-wiki/BootOptions/#boot-config-files

I am not aware of any kexec_menu.txt, where selected default boot option will be the option selected for default boot, being numbered (kexec_default.1.txt) which contains the default boot option parameters for that grub numbered entree.

Table entrees:
additions ?
argument removals ?
The table is screwed and passes into a new row last words of last row column.

Reply:
@tlaurion This is taken straight from https://github.com/osresearch/heads-wiki/blob/ef46a10c28f25a3260f24646a4bad85772d1659f/Installing-and-Configuring/install-os.md

I just moved it into a separate file.  I also do not understand your last sentence.